antora/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022-2023 The Khronos Group Inc.
+# SPDX-License-Identifier: CC-BY-4.0
+
+:chapters:
+
+* xref:index.adoc[Vulkan Guide]
+* Logistics Overview
+** xref:{chapters}what_is_vulkan.adoc[]
```

Comment:
Does this mean we need to add a new line here as well whenever a new file/chapter is added?

Reply:
Yes. This is obviously amenable to massaging out of the toplevel README.adoc but some scripting will be required. At the rate chapters are being added it doesn't seem like a big burden though we might want to put a comment in the README to remind people?

Reply:
agree not the worst thing ever, but yes, a small blurb in `CONTRIBUTING.adoc` to remind people to add it here (and maybe a quick "why we are using Antora" for those who can't find this PR)

Reply:
Done in CONTRIBUTING.adoc.